src/lib/Elf.cc

```diff
@@ -73,37 +77,36 @@ Elf::Elf(std::string path, char** imagePointer) {
    */
 
   // Seek to the byte representing the start of the header offset table.
-  uint64_t headerOffset;
-  file.read(reinterpret_cast<char*>(&headerOffset), sizeof(headerOffset));
+  uint64_t e_phoff;
+  file.read(reinterpret_cast<char*>(&e_phoff), sizeof(e_phoff));
 
   /**
-   * Starting 54th byte of the ELF Header a 16-bit value indicates
-   * the size of each entry in the ELF Program header. In the `elf64_hdr`
-   * struct this value maps to the member `Elf64_Half e_phentsize`. All
-   * header entries have the same size.
+   * Starting from the 54th byte of the ELF Header a 16-bit value indicates
+   * the the size in bytes of one entry in the
+   * file's program header table; all entries are the same
+   * size. In the `elf64_hdr`
+   * struct this value maps to the member `Elf64_Half e_phentsize`.
    * Starting from the 56th byte a 16-bit value represents the number
-   * of header entries in the ELF Program header. In the `elf64_hdr`
+   * of program header entries in the ELF Program header. In the `elf64_hdr`
    * struct this value maps to `Elf64_Half e_phnum`.
    */
 
   // Seek to the byte representing header entry size.
   file.seekg(0x36);
-  uint16_t headerEntrySize;
-  file.read(reinterpret_cast<char*>(&headerEntrySize), sizeof(headerEntrySize));
-  uint16_t headerEntries;
-  file.read(reinterpret_cast<char*>(&headerEntries), sizeof(headerEntries));
+  file.read(reinterpret_cast<char*>(&e_phentsize), sizeof(e_phentsize));
+  file.read(reinterpret_cast<char*>(&e_phnum), sizeof(e_phnum));
```

Comment:
Again, these names may be accurate to linux but they make the code less readable. Given there are public getter functions for these anyway with the accurate-to-linux naming, I think reverting these variable names to be inline with what was here previoiusly is preferable

Reply:
See comment above